charts/kube-prometheus-stack/values.yaml

```diff
@@ -626,7 +626,9 @@ grafana:
       multicluster: false
     datasources:
       enabled: true
-      defaultDatasourceEnabled: true
```

Comment:
I don't think it's a good to change the variable name as this would mean the feature could break for users wich copied the whole values file.

To prevent a therefore major version bump of the chart i would keep it the way it is.

Reply:
Do you have another proposal?
`defaultDatasourceName: "prometheus-sample"`?

If this is a decent request, then perhaps it should follow the format used throughout the values file, even if it were to require a major version change.

Reply:
This is already a bit mixed up in the chart so i just would avoid to change things, which could break stuff.
In the end, i'm not a nmaintainer of the chart so they have to approve or not anyway.